ansible/roles/provision-files/vars/main.yml

```diff
@@ -45,12 +47,90 @@ girder_file_downloads:
       sha512: 85bd2c6414cb582eba6e969726a7dc289644fb06e8b112e1fd1833351671ad701e0bdac1339c7429e4963f0058affa26fce7d7dd742bba854ad0e015085f576b
       dest: "{{ stage_dirs.activity_classifier }}/yolo_all_recipes_additional_objs_bkgd_act_mapping.txt"
 
-     # Global Step predictor model -- v2.0_sample_rate_2
+    # Global Step predictor model -- v2.0_sample_rate_2
     - file_id: 6543ca0a68ef53152cbab68d
       sha512: e9cc7c7e385700acfa1e2273b18346e6c85db8e9e49fc4ccea7686f3ac1e663086209b9268bc65e8f993f37a3436cb39e6a4da0ccc06cab0e8cdc795200eaaf1
       dest: "{{ stage_dirs.task_monitor }}/global_step_predictor_act_avgs_all_classes_v2.0_sample_rate_2.npy"
 
+    # ======== MEDICAL ========
+    # Pose estimation
+    - file_id: 6605cad48b763ca20ae99f7a
+      sha512: 50def808463afbeac8c2cb28dfb92ec0b45eb57672dc771d443ce89b2f94a938411328b79fdb25e33977db370922b2d3126c31d028f0caeb923209e2a4028c65
+      dest: "{{ stage_dirs.pose_estimation }}/pose_det_model.pth"
+    - file_id: 6605cc438b763ca20ae99f7d
+      sha512: 7ded5cb71ef0efe2444a19c7452073c63a34d65e3038b24236b304836abd47e8911e3d3023d2ba7e5a8d8c1e337e6f4b8e3080709827eb2b0daaedb0b6248561
+      dest: "{{ stage_dirs.pose_estimation }}/pose_model.pth"
+    # Hand detector
+    - file_id: 6605ca2e8b763ca20ae99f77
+      sha512: 42b56f34c0c443ad00d71e66334d6e852a7811011b22a60da9b5e2721565c21d1b5daad17c8a708908c9658ded81a663729090526feb5286bd0a591cf4fb5d8f
+      dest: "{{ stage_dirs.object_detector }}/hands_model.pt"
+    
+    # ---- M2 ----
+    # Object detector
+    # TODO
```

Comment:
@periakiva  What object detector is used for M2?

Reply:
uploaded it to data.kitware.com